programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+  let answer = "";
+  if (s.length % 2 == 0) {
+    answer = s.substr([s.length / 2 - 1], 2);
+  } else {
+    answer = s[parseInt(s.length / 2)];
+  }
+  return answer;
+}
```

Comment:
문자열을 배열처럼 사용할 수 있다는 걸 알고 계시는군요! 훌륭합니다...

이 전에 삼항 연산자를 사용하신 것처럼 이 문제도 삼항 연산자를 사용할 수 있습니다

```suggestion
function solution(s) {
   return (s.length % 2 === 0) ? s.substr([s.length / 2 - 1], 2) : s[parseInt(s.length / 2)];
}
```
추가 ) ==보다는 ===의 사용을 권장합니다.